src/Sulu/Bundle/ContactBundle/Entity/SocialMediaProfileType.php

```diff
@@ -0,0 +1,145 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\ContactBundle\Entity;
+
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\Common\Collections\Collection;
+use JMS\Serializer\Annotation\ExclusionPolicy;
+use JMS\Serializer\Annotation\Groups;
+use JMS\Serializer\Annotation\SerializedName;
+use JMS\Serializer\Annotation\VirtualProperty;
+
+/**
+ * Type of social media profile (i.e. Facebook,...).
+ *
+ * @ExclusionPolicy("All")
+ */
+class SocialMediaProfileType implements \JsonSerializable
+{
+    const TYPE_FACEBOOK = 'social_media_profile.facebook';
+    const TYPE_TWITTER = 'social_media_profile.twitter';
+    const TYPE_INSTAGRAM = 'social_media_profile.instagram';
```

Comment:
This way is quite hard coded, and not extensible at all 😕 Don't you want to put that into the configuration?